### PR TITLE
az: add page

### PR DESCRIPTION
**Pull request by Aracki**
I have added `az` page. It is an official Microsoft Azure CLI tool.

### CODE REVIEW

File: pages/common/az.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# az
+
+> The official CLI tool for Microsoft Azure.
+
+- Login in to Azure:
```
Comment (sbrl) [inline pages/common/az.md]: Did you mean this?

```suggestion
- Login to Azure:
```

File: pages/common/az.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`az account`
+
+- Manage Azure Kubernetes Services:
+
+`az aks`
+
+- Display one or many resources:
+
+`az get`
```
Comment (sbrl) [inline pages/common/az.md]: How do I specify the resource to get?

File: pages/common/az.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- Get all Azure Managed Disks:
+
+`az disk list`
+
+- Get all Azure virtual machines:
```
Comment (sbrl) [inline pages/common/az.md]: Perhaps this will read better?

```suggestion
- List all Azure virtual machines:
```

File: pages/common/az.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+- Manage Azure Kubernetes Services:
+
+`az aks`
+
+- Display one or many resources:
+
+`az get`
+
+- Get all Azure Managed Disks:
```
Comment (sbrl) [inline pages/common/az.md]: Perhaps this will read better?

```suggestion
- List all Azure Managed Disks:
```

File: pages/common/az.md (hunk 1 above)
Comment (pxgamer) [reply pages/common/az.md]: Or `Log in to Azure:`

File: pages/common/az.md (hunk 3 above)
Comment (Aracki) [reply pages/common/az.md]: Done!

File: pages/common/az.md (hunk 4 above)
Comment (Aracki) [reply pages/common/az.md]: Done!

File: pages/common/az.md (hunk 2 above)
Comment (Aracki) [reply pages/common/az.md]: This one example was a mistake.

File: pages/common/az.md (hunk 1 above)
Comment (Aracki) [reply pages/common/az.md]: `Log in` is better in this case.

File: pages/common/az.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- Log in to Azure:
+
+`az login`
+
+- Manage azure subscription information:
+
+`az account`
+
+- List all Azure Managed Disks:
```
Comment (mfrw) [inline pages/common/az.md]:
```suggestion
- List all Azure managed disks:
```

File: pages/common/az.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+`az disk list`
+
+- List all Azure virtual machines:
+
+`az vm list`
+
+- Manage Azure Kubernetes Services:
```
Comment (mfrw) [inline pages/common/az.md]:
```suggestion
- Manage Azure kubernetes services:
```

File: pages/common/az.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+`az aks`
+
+- Manage Azure Network resources:
```
Comment (mfrw) [inline pages/common/az.md]:
```suggestion
- Manage Azure network resources:
```

File: pages/common/az.md (hunk 5 above)
Comment (cam8001) [reply pages/common/az.md]: These are product names, and are capitalised that way in official documentation: https://docs.microsoft.com/en-us/azure/virtual-machines/windows/managed-disks-overview

Comment (mfrw) [reply pages/common/az.md]: ohh.. my bad! LGTM then

Comment (Aracki) [reply pages/common/az.md]: Yes; It is the case in all examples.

**Comment by sbrl** [comment]
Are we ok to merge, @mfrw?

**Pull request merged**